app/javascript/controllers/map/tokens_controller.js

```diff
@@ -50,6 +50,9 @@ export default class extends Controller {
 
   startMoveToken(event) {
     event.stopPropagation()
+    if (this.hasDrawerTarget) {
+      this.drawerTarget.dispatchEvent(new CustomEvent('open'))
```

Comment:
How do you feel about these events being defined elsewhere as constants? Or the event names?

Only asking as it seems to be the practice for similar use in Redux

Reply:
Yeah, the thing that is interesting to me is that we have the ToggleController, so it feels like the ToggleController should actually have those constants or otherwise be more encapsulated, but I did some reading and emitting events seems to be the recommended way with stimulus.

Do you think they should live in constants.js file or something?

Reply:
From reading through the rest of the PR, these events are used as the `data-action` event types so it might be that we're not really getting much extra value from extracting them. They'd need to be available to the renderers to make it worthwhile. So all I have is maybe pulling them out to some YAML file.

Let's see where else we use custom events before worrying about that then.